rs/ic_os/config/src/types.rs

```diff
@@ -148,6 +147,7 @@ pub struct DeterministicIpv6Config {
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct FixedIpv6Config {
-    pub address: Ipv6Addr,
+    // fixed ipv6 address includes subnet mask /64
+    pub address: String,
```

Comment:
For fixed ipv6 address, the address will contain the subnet mask, so it's easier to store as a String rather than an Ipv6Addr